### DISCUSSION TITLE
Budibase license

### DISCUSSION THREAD

**Discussion by mamwadei** [Budibase/budibase#12723] [Help] [answered]
I would like to use budibase for my non profit org. But I hear if I use it for free it is only limited to 5 users  even on premises deployment. Is that right.?

 If you can tell what constraints that it has in the context?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @mamwadei,

The free limit is 20 for local and cloud installations.

Feel free to contact the sales team on the website [here](https://budibase.com/contact/). A member of the sales team can talk through your use case with you and see if theres deals or discounts that can be provided.
